src_py/camera.py

```diff
@@ -13,8 +13,10 @@ def init():
     use__camera = True
 
     if sys.platform == 'win32':
-        use_vidcapture = True
-        use__camera = False
+        if sys.version_info > (3,):
```

Comment:
Probably need a version check here for >= Windows 10 (if that is a requirement?)

Reply:
I was looking at checking that in C, but I was getting confused by all the Windows stuff about pretending to be an earlier version for apps that don't have yada yada in their manifest.

But this looks like a good solution, thanks.